storage/leveldb/leveldb.go

```diff
@@ -80,7 +88,11 @@ func NewDB(path string, batchDelaySeconds int, maxBatchSize int, maxOpenFiles in
 	})
 
 	crtCounter := atomic.AddUint32(&loggingDBCounter, 1)
-	log.Debug("opened level db persister", "path", path, "created pointer", fmt.Sprintf("%p", bldb.db), "global db counter", crtCounter)
+	sw.Stop("NewSerialDB")
```

Comment:
"NewDB" is started, but "NewSerialDB" is stopped. Maybe use a variable to be sure the same sw is started and stopped

Reply:
good catch, refactored.